proposals/solid-notifications.md

```diff
@@ -0,0 +1,270 @@
+# Solid Notifications
+
+**Status**: Proposal
+
+## Introduction
+
+Much of the Solid Protocol is built on a RESTful API. This allows Solid to make use of well-understood interactions with resources on the Web, and this satisfies many use cases for client applications in the Solid ecosystem. However, for interactive client applications with multiple participants, chat-apps being just one such example, a RESTful API is too limited, as clients need to rely on polling in order to discover changes in the underlying data.
+
+A notification API for Solid would complement the core RESTful API so that clients can listen for updates to particular resources. In addition, a notification-based mechanism may reduce latency and load on a Pod server.
+
```

Comment:
Add a mention that existing Sold servers and clients implementing a notification protocol based on web sockets, and this being used for example for chat applications, collaborative editors, etc.  It is disingenuous to write as though this is not currently already running.  This protocol serves to provide a more secure and more modular version of the same functionality.

Reply:
Noted. The introduction text will likely be completely rewritten. Does this concern block merging this PR in its present form?

Reply:
#12